release-notes/opendistro-for-elasticsearch-sql.release-notes-1.12.0.0.md

```diff
@@ -1,6 +1,14 @@
 ## 2020-11-20 Version 1.12.0.0
 
 ### Features
+* Add count() support for PPL ([#894](https://github.com/opendistro-for-elasticsearch/sql/pull/894))
+* Add like, isnull, isnotnull function in PPL ([#893](https://github.com/opendistro-for-elasticsearch/sql/pull/893))
+* Revert Error Messages for Run/Explain ([#889](https://github.com/opendistro-for-elasticsearch/sql/pull/889))
+* Extend the PPL identifier defintion ([#888](https://github.com/opendistro-for-elasticsearch/sql/pull/888))
+* Add identifier and datatype documentation for PPL ([#873](https://github.com/opendistro-for-elasticsearch/sql/pull/873))
+* Add Error Message for Explain and Run ([#872](https://github.com/opendistro-for-elasticsearch/sql/pull/872))
+* Revert java version in jdbc release workflow ([#871](https://github.com/opendistro-for-elasticsearch/sql/pull/871))
```

Comment:
remove this?

Reply:
Should this be removed? Or just moved to bug fix section?

Reply:
I think we'll need to include all merged PRs, but I'll move it to infra section, thanks!